tools/hangar/frontend/src/components/tabs/ServerTab.tsx

```
@@ -1629,6 +1656,8 @@ function PythonCell({
   onSettings,
 }: {
   running: boolean;
+  /// Host LAN address, or "" when unknown — see the fetch in ServerTab.
```

ESLint

[error] Expected exception block, space or tab after '//' in comment.

(spaced-comment)